tests/unit/helpers.js

```diff
@@ -431,3 +433,25 @@ export const getFakeConfig = (params = {}) => {
   }
   return Object.assign(configUtil.cloneDeep(config), params);
 };
+
+/*
+ * A sinon matcher to check if the URL contains the declared params.
+ *
+ * Example:
+ *
+ * mockWindow.expects('fetch').withArgs(urlWithTheseParams({ page: 1 }))
+ */
+export const urlWithTheseParams = (params) => {
+  return sinon.match((urlString) => {
+    const { query } = url.parse(urlString, true);
+
+    // eslint-disable-next-line no-restricted-syntax
+    for (const param of Object.keys(params)) {
```

Comment:
If we're doing a loop anyway wouldn't it be a better habit to just do `for (const param in params) {` which doesn't create a new array? We can already iterate through params. I get you'll want to check for `hasOwnProperty` then, I guess it's up to you 🤷‍♂️ 

Your way looks nicer but is I guess less efficient. That probably doesn't matter for the tests much but I don't think this is a case where the faster option is harder to read/reason about.